js/src/carousel.js

```diff
@@ -120,6 +120,7 @@ class Carousel extends BaseComponent {
     this._touchSupported = 'ontouchstart' in document.documentElement || navigator.maxTouchPoints > 0
     this._pointerEvent = Boolean(window.PointerEvent)
 
+    this.isRTL = this._element.dir === 'rtl' || isRTL
```

Comment:
Why is the new check needed? Or why doesn't the previous one suffice?

BTW, I have a draft PR around to switch `isRTL` to a function so that it's tree-shaken, but I doubt it's related.

Reply:
I had to add it so that I can produce the unit tests I was asked for. Then I realized it might be helpful in some cases.

isRTL only checks if the <HTML> tag has dir=rtl attribute. This means that you can't have an RTL carousel otherwise. One use case: in the documentation for bootstrap itself on how to make a page rtl. The page itself could be in English, but the carousel could be in Arabic.

Reply:
I don't think we support such a case in v5 @ffoodd ?

Reply:
Yes, that case is not supported yet. And this check is then redundant :slightly_smiling_face:

Reply:
@AqeelAT: I merged #32446. Your use case makes sense, but I thought it wasn't supported.

That being said, it seems it's sort of supported https://getbootstrap.com/docs/5.0/getting-started/rtl/#ltr-and-rtl-at-the-same-time, so after #32913 or in  #32913, we could refactor the isRTL function to check for an element's direction and or `document.documentElement.dir`.

We just need to make sure we have JS tests to cover these changes